node/mock/outportStub.go

```diff
@@ -0,0 +1,78 @@
+package mock
```

Comment:
A bit of duplicated stubs and mocks for the outport & outport driver. Optionally, if helpful in this case, might be moved to `testscommon/outport`?

Reply:
done.